.editorconfig

```diff
@@ -0,0 +1,234 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+indent_style = space
+tab_width = 4
+
+# New line preferences
+end_of_line = crlf
+insert_final_newline = true
+
+#### .NET Coding Conventions ####
+
+# Ignored analyzer rules
+dotnet_diagnostic.CA1725.severity = none # Interface implementations should use same parameter names
+
+# Organize usings
+dotnet_separate_import_directive_groups = false
+dotnet_sort_system_directives_first = false
+file_header_template = unset
```

Comment:
Can the file header be set to our copyright header?

```
﻿// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
//
```

Reply:
Yes- good call

Reply:
Added in [976bbe8](https://github.com/microsoft/FeatureManagement-Dotnet/pull/436/commits/976bbe85538ae39e016907ac94cff0a9fba2180b)